### PR TITLE
feat: Add "Reply-to" to organizers in session notify

### LINKED ISSUE #6544: Notify Speakers: Change "Reply-to" to actual organizers instead of eventyay
The first iteration of the submission system used eventyay.com emails. This was good to get it up and running. 

Moving to the next iteration we now need this feature to specifically support communication with organizers. Speakers who receive an email from the submission system should no longer reply to eventyay but instead directly to the organizers of the event.

Therefore please change the following:

* Change the email from/reply as to the organizers of the event. We can show their email addresses here as the organizers should know their own email. Also to enable communication of speakers with organizers their email addresses can be shared here. Please implement this as follows
a) show the info on the notify speakers pop up
b) enter this following as default for sending the email in the code

From: eventyay submission system <submissions@eventyay.com>
Reply-to: Event Organizer 1 <organizer1email@orgdomain.com>, Event Organizer 2 <organizer2email@orgdomain.com> ...

* Below text box add text "A copy of this email will be send to all organizers of this event."

Screenshot from 2021-01-30 13-34-30
Screenshot from 2021-01-30 13-34-04

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes #6544 

new pr 4
new pr

### CODE REVIEW

File: app/components/modals/session-notify-modal.hbs [hunk 1]
```diff
@@ -11,13 +11,29 @@
   {{#if this.mail}}
     <form class="ui form">
       <div class="field">
-        <label>Name and Email</label>
+        <label>{{t 'From'}}</label>
         <Input
           type="text"
-          name="name and email"
+          name="from"
+          @value={{t 'eventyay submission system <submissions@eventyay.com>'}}
```
Comment (iamareebjamal) [inline app/components/modals/session-notify-modal.hbs:18]: What is eventyay? This project is open event

**Comment by iamareebjamal** [comment]
This is just the UI, where is the logic for reply to?

**Comment by sachinchauhan2889** [comment]
https://github.com/fossasia/open-event-server/pull/7714

**Comment by iamareebjamal** [comment]
OK

File: app/components/modals/session-notify-modal.js [hunk 2]
```diff
@@ -42,6 +42,14 @@ export default class SessionNotifyModal extends ModalBase {
     return session.speakers.map(speaker => `${speaker.name} <${speaker.email}>`).join(', ');
   }
 
+  @computed('sessionId')
+  get organizersEmails() {
+    const session = this.store.peekRecord('session', this.sessionId);
+    const owner = session.event.get('owner');
+    const ownerEmail = owner.get('fullName') + ' <' + owner.get('email') + '>';
+    return session.event.get('organizers').map(organizer => `${organizer.fullName} <${organizer.email}>`).concat(ownerEmail).join(', ');
```
Comment (iamareebjamal) [inline app/components/modals/session-notify-modal.js:50]: Is this public info? People can see organizers and their email?

Comment (iamareebjamal) [reply app/components/modals/session-notify-modal.js:50]: If so, it is a bug, please open an issue on the server

File: app/components/modals/session-notify-modal.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ export default class SessionNotifyModal extends ModalBase {
   @tracked subject = '';
   @tracked message = '';
   @tracked bccString = '';
+  @tracked settings = this.modelFor('events.view.sessions.list').settings;
```
Comment (iamareebjamal) [inline app/components/modals/session-notify-modal.js:17]: No need for tracked. Use settings service

File: app/routes/events/view/sessions/list.js [hunk 4]
```diff
@@ -75,14 +75,17 @@ export default class extends Route.extend(EmberTableRouteMixin) {
 
     const sessionStatesMapPromise = sessionStateMapCached || this.loader.load('sessions/states');
 
-    const [feedbacks, sessions, sessionStateMap] = await Promise.all([feedbacksPromise, sessionsPromise, sessionStatesMapPromise]);
+    const settingsPromise = this.store.queryRecord('setting', {});
```
Comment (iamareebjamal) [inline app/routes/events/view/sessions/list.js:78]: Why? You need to check your PRs before requesting review if they are ready or contain unneeded changes

**Comment by mariobehling** [comment]
Well-done. 

There is are two issues though: 
1. If there are more organizers, it does not show all reply-to addresses as the field does not change to several lines.
2. If an organizer gets deleted from an event, the organizer still shows up as a reply-to address even though the organizer was already deleted.

**Comment by mariobehling** [comment]
So, there is another issue happening after this PR.

Before the PR a copy of the email was sent to the organizer email. After the update the copy of the email does not go to the organizer email anymore.

**Comment by sachinchauhan2889** [comment]
@iamareebjamal  @mariobehling sir, i have checked out my PR at server side. My PR only adds reply-to field in mail. 
I removed changes(done by my merged PR) in my local setup and checked email still not send to all organizers. 
The only difference I am able to see is reply-to field. There is no CC containing all organizers in mail, neither mail send by looping all organizers. 

My observation :- 
## before

before

## after
after

**Comment by iamareebjamal** [comment]
Yes, this email was never sent to organizers. Even before this PR

The organizer could manually add bcc if they wanted, but it was not automatically done

**Comment by mariobehling** [comment]
Ok, thanks for the info. So, basically before a copy of the email was received by office@fo... and there is no more copy arriving. Let's discuss how we want this to work later. Good job looking into detail! Thanks.

**Pull request merged**